content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,324 @@
+---
+title: Package and deploy your models with MLEM
+date: 2022-05-24
+description: >
+  Introducing MLEM -- Machine Learning Engineering Management for Model
+  Deployment and Management.
+descriptionLong: >
+  We’re excited to announce the launch of our latest open source offering,
+  [MLEM](https://mlem.ai)! MLEM is a tool that automatically extracts meta
+  information like environment and frameworks from models and standardizes that
+  information into a human-readable format within Git. ML teams can then use the
+  model information for deployment into downstream production apps and services.
+  MLEM easily connects to solutions like Heroku to dramatically decrease model
+  deployment time.
+picture: 2022-05-24/mlem-rocket.png
+author: aguschin
+# commentsUrl: TODO
+tags:
+  - Machine Learning
+  - Serving
+  - Deployment
+  - Model Registry
+  - MLOps
+---
+
+We built MLEM to address issues that MLOps teams have around managing model
+information as they move them from training and development to production and,
+ultimately, retirement. MLEM is meant to help teams automate the collection of
+information around how the model was trained, what the model is for, and
+operational requirements around deployment.
+
+Just like all our [other](https://dvc.org) [tools](https://cml.dev), MLEM uses
+your Git service to store model information and connects with CI/CD solutions
+for deployment (like Heroku). This Git-based model
+([one of our core philosophies](https://iterative.ai/why-iterative/)) aligns
+model operations and deployment with software development teams – information
+and automation is all based on familiar DevOps tools – so that deploying any
+model into production is that much faster.
+
+Capturing model-specific information requires the understanding of the
+Programming language and ML frameworks they're created with. That's why MLEM is
+a Python-specific tool. To provide developer-first experience, MLEM exposes
+carefully designed CLI to help you manage DevOps parts of the workflow from CLI
+and Python API to handle model productionization programmatically.
+
+With MLEM, ML teams get:
+
+- **Model metadata codification**: Human-readable information about a model for
```

Comment:
it looks like codification is only for search and docs, but this undersells it. This meta-information is needed to deploy things in the first place, to build clients faster, etc? This is main purpose. 

Ideally we can converge this into a single value prop - packaging models to deploy, everything else comes as a benefit on top? 

otherwise we start with some philosophy, then we go into codification ... and only after we go into deployment ... and only after into model registry ... it feels it should be presenting things other way around - high level solution / value prop first, then goes into impl details and ... or at least they should come really close to each other

I hope it makes sense :) happy to brainstorm on this more if needed ...

Reply:
If "codification" is to niche or technical maybe speak of the user benefits like "reliable, standard metadata".

Reply:
Thank you for your review, @shcheklein!
1. It seems I can't distribute the first section paragraphs ("We built MLEM to address..." and "Just like all our..." and "Capturing model-specific...") anywhere except for the second section "Model metadata codification". At least in the current form. So I can try to rewrite those and move them to the rest of the document. But after addressing your other comments that may be not needed anymore. Please let me know WDYT.
2. I think we need to use "codify". It sounds great and explains what MLEM does with metainformation in a single word - that's good for the quicker explanations later. I've provided some description about codification right after the first word occurence. Do you think it's enough?
3. Removed DVC code examples.
4. I think this is addressed now.
5. "Productionize your models with MLEM in a Git-native way" maybe?
6. I put a picture with a dog asking for the stars for now :)